docs/ACCESSIBILITY.md

```diff
@@ -293,6 +293,46 @@ To make the live region more consistent across browsers and easier to control, w
    -  Only change focus synchronous to user gesture
    -  Related to [#3135](https://github.com/microsoft/BotFramework-WebChat/issues/3135)
 
+# Controlling the narration of activities and attachments
+
+> This is related to [#3360](https://github.com/microsoft/BotFramework-WebChat/issues/3360), [#3615](https://github.com/microsoft/BotFramework-WebChat/issues/3615), [#3918](https://github.com/microsoft/BotFramework-WebChat/issues/3918).
+
+## User story
+
+A bot developer want to set the narration for a message activity. The activity may or may not have attachments.
+
+For example, it is required for the following user stories:
+
+-  The message contains Markdown
+   -  For example, the `text` field is `"Hello, *World!*"`
+   -  Desirable: "hello world"
+   -  Undesirable: if `speak` field is not set, it will be narrated as "hello (pause) asterisk world asterisk"
+-  The message contains HTML
+   -  For example, the `text` field is `"## Exchange rate:\n\n<table><tr><th>USD</th><td>1.00</td></tr><tr><th>JPY</th><td>0.91</td></tr></table>"`
+   -  Desirable: "Exchange rate for 1 USD dollar is 0.91 Japanese yen."
+   -  Undesirable: any HTML or Markdown syntax
+-  The message contains a document, such as an insurance policy
+   -  For example, the `text` field is `"Insurance policy:"`, and the attachment contains a file named `12345678-1234-5678-abcd-12345678abcd.doc`
+   -  Desirable: "The insurance policy is ready to download."
+   -  Undesirable: any narration containing the bogus file name
+
+## Implementation
+
+Based on [Bot Framework Activity spec](https://github.com/microsoft/botframework-sdk/blob/main/specs/botframework-activity/botframework-activity.md), the following fields can be used to control the narration: `speak` and `text`.
+
+We implemented the following logic for computing the text for screen reader:
+
+1. If `speak` field present
+   1. If `speak` field is not an empty string, narrate the field, don't narrate attachments
```

Comment:
Empty string, or space?

Reply:
Empty string for `webchat:fallback-text`.